enos/modules/build_artifactory_package/main.tf

```diff
@@ -73,12 +68,11 @@ module "artifact_metadata" {
 }
 
 data "enos_artifactory_item" "vault" {
-  username = var.artifactory_username
```

Comment:
Before merge we probably ought to test the dev scenarios with the `package:rpm` and `package:deb` variants to make sure they're working as expected.